google/cloud/aiplatform/vizier/study.py

```diff
@@ -88,7 +88,7 @@ def __init__(
     def create_or_load(
         cls,
         display_name: str,
-        problem: vz.ProblemStatement,
+        problem: vz.StudyConfig,
```

Comment:
Looks like `ProblemStatement` is imported in `pyvizier/__init__.py`. Is the class no longer being used and its features are now completely replaced and/or incapsulated in `StudyConfig`?

Reply:
The ProblemStatement is the rename of Study config.

Reply:
ProblemStatement is the base class for StudyConfig, so both can be used from a pytype perspective.
Using StudyConfig here means that we want the Vertex specific StudyConfig, and not any subclass of ProblemStatement.

We expect Vertex users to only create StudyConfig objects, not ProblemStatement objects.